pallets/parachain-staking/src/lib.rs

```diff
@@ -716,6 +717,19 @@ pub mod pallet {
                 selected_collators_number: v_count,
                 total_balance: total_staked,
             });
+
+            log::info!(
+                "Staking storage chain/current storage version: {:?} / {:?}",
+                Pallet::<T>::on_chain_storage_version(),
+                Pallet::<T>::current_storage_version(),
+            );
+
+            // Set storage version
+            log::info!(
+                "Setting staking storage to latest version: {:?}",
+                crate::migration::STORAGE_VERSION
+            );
```

Comment:
Do we need this? We can query the value from state if we need to.

Reply:
I think its cleaner if we set the version of the state, when we build the genesis state.
The original issue we are experiencing is because this was missing.

Reply:
I updated the logs a bit, and reduced the level.

Reply:
I updated the logs a bit, and reduced the level.

Reply:
I agree about setting the version on genesis. I meant about the logs. I think we can remove the logs as they don't add much value.